### ISSUE TITLE
Request design review for VSA Authenticated Experience Profile 2.0

### ISSUE THREAD

**Issue by tressaellen** [department-of-veterans-affairs/va.gov-team#5061]
Previous research showed the need to combine the 'Profile' and 'Account' pages into one singular unit #2086 .  This is part of a larger issue which can be reviewed under the [Personalization 2.0](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Personalization/Personalization%202.0/Discovery%20&%20Research/Personalization%202.0%20Discovery%20Summary%20&%20Strategy.md) 

The goals of this endeavor are to:

- Reduce redundancy — Specifically, we should reduce redundancy between the My VA dashboard and the VA.gov homepage.

- Get users where they need to go — We should balance bubbling up content with showing a succinct list of links to quickly route people to tools.

- Elevate personally relevant information — Users should see themselves in these updates.

- Scale gracefully — Our designs should allow for future updates and expansion.

- Prioritize wayfinding — We should make it easy for users to find information that is relevant to them, and we should guide users accordingly depending on where they are in their journey at the VA.

- Effectively use visual space — We should avoid a single-column, "content page" approach. Design updates should more effectively employ the design system and update it as needed.

These are the most recent designs - but I am currently in the process of making them more scalable so they are subject to some changes.

*EDIT* I've attached two more screens to address the concern that tabs might not be as scalable. For Profile, we believe that in future states there will be a max of 2 extra tabs needed to be added.  These pages have different UI designs, but address the same UX issue. (Pages 2 and 4 attached at the end)

![Mobile 1](https://user-images.githubusercontent.com/58752772/72923032-74b25400-3d0b-11ea-87b7-e8d0a2930447.jpg)
![Mobile 2](https://user-images.githubusercontent.com/58752772/72923034-74b25400-3d0b-11ea-97f9-eec96ed8ab39.jpg)
![Mobile 3](https://user-images.githubusercontent.com/58752772/72923035-754aea80-3d0b-11ea-89ae-682cf0ae733e.jpg)
![Profile 2 0](https://user-images.githubusercontent.com/58752772/72923120-a0353e80-3d0b-11ea-8d56-d0c3dee4e6fc.jpg)

![2](https://user-images.githubusercontent.com/58752772/73020477-ef9a6e00-3de2-11ea-90be-189569027469.jpg)
![4](https://user-images.githubusercontent.com/58752772/73020478-ef9a6e00-3de2-11ea-90bb-2804fbf9dc17.jpg)

**Comment by Samara-Strauss** [comment]
Hey all — we have yet to request a design review via this format. Is this something we throw over the wall to you, and then you throw back? Or is this something we'd have a meeting about?

**Comment by CrystabelReiter** [comment]
Hi Samara! With a design review in this format we typically schedule a 30 minute meeting where you'll spend the first 10 minutes presenting your work and the rest of the meeting is spent on giving initial feedback then we asynchronously provide feedback on this ticket for both of you to read later. We also typically invite other practice areas like IA & Content as well

**Comment by Samara-Strauss** [comment]
Thanks @CrystabelReiter! Feel free to schedule this review with Tressa whenever you guys have time, and I'll make it if I can.

**Comment by CrystabelReiter** [comment]
For ease of scheduling, VSP has dedicated office hours next week for reviewing on Tuesday 11am-12pm EST, and Thursday 1:30-2:30 PM EST. Please let us know which 30 minute slot works best for you two @Samara-Strauss @tressaellen

**Comment by jenstrickland** [comment]
Both accessibility specialists will be at an off-site on Tuesday, January 28th, so may need to have a separate review. Apologies for the inconvenience! This is a rarity.

**Comment by Samara-Strauss** [comment]
I'll let @tressaellen schedule things, but the sooner the better! We're trying to get the design direction finalized before the end of this sprint.

**Comment by tressaellen** [comment]
@CrystabelReiter let's go ahead and do Tuesday 11-12 EST.  @jenstrickland if you have time before then for a quick informal review, that would be great!

**Comment by mnorthuis** [comment]
Early IA directional feedback...IA will be reviewed in detail and finalized once user-testing is complete and designs are more closely solidified. 

- From an IA perspective, there are no issues with the tabs vs a left nav design for navigating different sections of the profile.  If a tabbed version is used and there's a concern that the number of tabs may exceed the width of the screen, I would review how the content within profile is organzied and challenge design to reduce the number of sections/groupings. 
- A card sort activity could be helpful in determining the tabs/groups, what goes in them and labeling
- Be sure tab labels, although shortened, still have enough information scent for users to understand what they would find in the section
- Ensure we have the ability to link directly to profile with a specific tab defaulted to open (by way of parameters or separate URL) - i.e. a user may go from the unauthenticated static page for direct deposit and be dropped directly into that tab in the profile.  This experience should be tested as well to ensure its not disorienting to users and they understand where they were taken. 
- Breadcrumbs are currently based on page hierarchy, so unless you are going to a different page, the current page breadcrumb label does not change. In other words, the breadcrumb will always say "Home > Profile". If user testing shows users want to use the breadcrumb to navigate to the default Profile view, we can/should revisit this.

**Comment by Samara-Strauss** [comment]
Thanks for this @mnorthuis! All of this sounds good to me.

> Ensure we have the ability to link directly to profile with a specific tab defaulted to open

Contact Information should be the first section. We can have /profile redirect to /profile/contact-information. Each section should have its own URL under /profile for easy linking.

**Comment by CrystabelReiter** [comment]
Early design feedback from Emily and me: 

-  Testing out a vertical and horizontal layout would be beneficial to figure out how people like to read their information
- We recommend changing the exclamation point icon to the information icon and color it black to make sure it doesn't seem clickable 
- In the header, where the additional info component is, the dashed line could use a color that has more contrast
-For the tabs, keep in mind smaller screens like tablets or smaller laptops and how that would look along with the hover and click states.

**Comment by Samara-Strauss** [comment]
> Testing out a vertical and horizontal layout would be beneficial to figure out how people like to read their information

Is this specifically re: navigation or something else? 

Also, great feedback all around! Thank you all for your help :D

**Comment by CrystabelReiter** [comment]
Specifically the form fields for addresses, phone numbers, and email addresses. 

This is great work!

**Comment by Samara-Strauss** [comment]
> Specifically the form fields for addresses, phone numbers, and email addresses.

Noted! Thanks for the clarification. We should definitely test for this.

As an aside, I think I want us to test the mobile view of this primarily and then show desktop views later in user testing. My biggest concerns are the mobile nav and will people know how to navigate through the profile in the mobile view? But we should also have them look at desktop so we can test different elements of that layout as well.

**Comment by emilywaggoner** [comment]
@Samara-Strauss I have wondered similar things about the mobile nav! I think that would be informative across the board, and could potentially lead to some design initiatives specific to that global component.

**Comment by tressaellen** [comment]
I made a few small changes to the cards/layout/tabs according to feedback - just wanted to post here for documentation. 

![4](https://user-images.githubusercontent.com/58752772/73405136-6a1a3080-42b0-11ea-9d52-0310b4e47b00.jpg)

**Comment by Samara-Strauss** [comment]
This is so solid!

Quick q — where did the checkbox go for health care reminders?

**Comment by tressaellen** [comment]
@Samara-Strauss I was under the impression that was for future builds - but if it's for this round then I can add it back in no problem

**Comment by Samara-Strauss** [comment]
@tressaellen you are correct the feature has not launched yet, but since it is in active development and hopefully close to launching, let's put it back in.

**Comment by jenstrickland** [comment]
_VSA Accessibility Notes_

- I didn't notice mobile Profile menu at all, and was confused when I saw the screen of it open. It wasn't obvious to me that those were links until I realized it was opened via the hamburger icon. Consider testing to see if this is obvious to users (I see now in responses that is the plan. 🥇 ). Recommend underlines for the links in the navigation, unless it tests well. [WCAG 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html) recommends color not be the only indicator of interactivity.

- For an experiment, grab an empty paper towel tube or roll a piece of paper up to a similar shape (Alternatively, see this video https://youtu.be/LeWAnR4JPM0, specifically at 25:00). View the mobile prototype, now view the desktop prototype. There are proximity considerations for desktop, which impact low-vision and lowered comprehension users. **Recommendation:** Consider affordances such as a rule or 'zebra-striping' to connect corresponding info. In the latest version, I am not sure the proximity issue is quite resolved yet.

- Is there a left-nav desktop version? I do love the tabs at the top, but I think if there's a left nav, then the proximity issues may be resolved.

- What happens when a person clicks Edit?

- How long is the content that is hidden under "How do I update the email used to sign-in to VA.gov? If it is a single sentence, consider replacing this sentence and design system's additional info component with the sentence. If it's longer than ~200 chars, carry on! :)

**Comment by tressaellen** [comment]
@jenstrickland thank you for the feedback! I'll take some time to go through these.

**Comment by jenstrickland** [comment]
@tressaellen One addition to my previous comments, after discussion with @rtwell - Identification of the h1 for these pages. I _suspect_ it would be `<h1><dfn>Your profile:</dfn> Full Name of Veteran/Service Member</h1>`. This brings up complexity since each of the individual pages listed in the sidenav is the `<h1>` if we follow the pattern used elsewhere. What would you say is the H1 for the pages within Profile 2.0.

**Comment by tressaellen** [comment]
@jenstrickland great point - I'll add a "Your Profile" above "Kim Washington" so we can be consistent. Thank you!
